### PR TITLE
Exercise one/ordering the words

### PR DESCRIPTION
**Pull request by Enrique199413**
# Description:

Given a string, you need to write a method that order every letter in this string in ascending order.

Also, you should validate that the given string is not empty or null. If so, the method should return:
`"Invalid String!";`

## Examples
```
new Ordering().orderWord("hello world") => " dehllloorw"
new Ordering().orderWord("bobby") => "bbboy"
new Ordering().orderWord("") => "Invalid String!"
```
## Notes

• the given string can be lowercase and uppercase.
• the string could include spaces or other special characters like '# ! or ,'

### CODE REVIEW

**Comment by jresendiz27** [comment]
resolve conflicts first

File: src/main/java/mx/ipn/jresendiz/codewars/OrderStrings.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                for (int number:OrderStrings.orderAscending(characteres)) {
+                    result += (char) number;
+                }
+
+            }
+        } catch (NullPointerException e) {
+            result = "Invalid String!";
+        }
+        System.out.println(result);
+        return result.toString();
```
Comment (jresendiz27) [inline src/main/java/mx/ipn/jresendiz/codewars/OrderStrings.java]: redundant code

**Comment by jresendiz27** [comment]
@Enrique199413 resolve the merge conflicts and all the issues listed above

**Comment by Enrique199413** [comment]
I wait for anything? :grin:

**Comment by jresendiz27** [comment]
create a new PR and add your code, it was reverted.

**Pull request merged**